version/version.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 // Version is the current provider main version
-const Version = "1.9.0"
```

Comment:
Can you rebase the branch we already upgraded to v1.10.0

Reply:
done